--- CMakeLists.txt	2016-12-21 16:49:39.797928710 +0100
+++ CMakeLists.txt	2016-12-21 16:50:31.879346315 +0100
@@ -24,6 +24,8 @@
 ################################################################################
 project(cegui)
 
+set (CMAKE_POSITION_INDEPENDENT_CODE ON)
+
 set (CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${PROJECT_SOURCE_DIR}/cmake")
 include(CEGUIMacros)
 include(CMakeDependentOption)
